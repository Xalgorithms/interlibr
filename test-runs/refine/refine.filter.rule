META
  VERSION "0.0.1"
  MAINTAINER "Don Kelly <karfai@gmail.com>";

REQUIRE refine:table0:0.0.1 AS table0;

REFINE table0 AS table1
  FILTER b < 10;
