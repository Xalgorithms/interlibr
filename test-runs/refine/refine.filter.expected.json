{
  "tables" : {
    "table" : {
      "table1" : [
        { "a" : 1, "b" : 2  },
        { "a" : 2, "b" : 4  },
        { "a" : 3, "b" : 9  }
      ]
    }
  }
}
